### DISCUSSION TITLE
Unraid Docker AIO Install permissions issue

### DISCUSSION THREAD

**Discussion by madman2012** [nextcloud/all-in-one#1956] [Questions] [answered]
I have a NextCloud AIO instance on a Unraid Server. Still, for some reason, I have to regularly reset permissions with "chmod -R 0770 /mnt/user/NextCloud\ AIO && chown -R 33:0 /mnt/user/NextCloud\ AIO" command to keep it from erroring out with "Please change the permissions to 0770 so that the directory cannot be listed by other users."

Cache is not enabled for the Unraid share and mover is disabled so it doesn't move the files since they are written directly on the array. 

I am lost on how to fix this other than regularly running the command. Everything is running fine otherwise.

Thank  you for your help in advance.

**Comment by szaimen** [comment, marked as answer]
Hi, unfortunately I dont know much about unraid. 

However to me it sounds like the permissions are externally changed e.g. due to writing or changing files not via Nextcloud diretly but instead via smb or else. Could that be the case here?

**Reply by madman2012** [reply]
Thanks. I disabled the SMB and will monitor for the next few days. Hopefully that's the case, if not I'll keep everyone posted! Thanks for your help!
